src/extract_sf.c

```diff
@@ -240,7 +243,14 @@ int run(char *in_filename, bool use_fmt, int nevn, int run_no) {
     }
 
     // Create output file.
-    TFile *f_out = TFile::Open(Form("../root_io/sf_study_%06d.root", run_no), "RECREATE");
+    char*  out_filename = (char *) malloc(128 * sizeof(char));
+    if(!use_simul){
+        sprintf(out_filename, "../root_io/sf_study_banks_%06d.root", run_no);
```

Comment:
This isn't working for data, there seems to be a mistake beforehand when saving the run_no variable.

Reply:
Also, I don't understand the reasoning behind the filename change? It used to be only `sf_study_%06d.root`

Reply:
Fixed the data problem. In the get_run_no() function I was using use_simul in the ifs instead of *use_simul....
The second point was a taldo. Reverted to the old name convention.

Reply:
Thanks!